### DISCUSSION TITLE
FZF: An error has occurred in 'editor.fzf + project.nvim'

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3621] [Q&A]
After I enabled the 'editor.fzf' plugin, Unable to use `project.nvim`.

```
E5108: Error executing lua: vim/_editor.lua:431: nvim_exec2(): Vim:E492: Not an editor command: Telescope projects
stack traceback:
	[C]: in function 'nvim_exec2'
	vim/_editor.lua:431: in function 'cmd'
	...im-data/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:24: in function <...im-data/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:20>
```

**Comment by dpetka2001** [comment]
It has been fixed on latest [HEAD](https://github.com/LazyVim/LazyVim/pull/3594). Just wait for the next stable release or switch to following HEAD to get the latest changes immediately.

**Reply by iamxiaojianzheng** [reply]
> It has been fixed on latest [HEAD](https://github.com/LazyVim/LazyVim/pull/3594). Just wait for the next stable release or switch to following HEAD to get the latest changes immediately.它已在最新的 HEAD 上修复。只需等待下一个稳定版本或切换到关注 HEAD 即可立即获得最新更改。

I also found an issue with the preview window occasionally not closing automatically when using fzf.

**Reply by iamxiaojianzheng** [reply]
like this.

![image](https://github.com/LazyVim/LazyVim/assets/29248764/b6f20c65-50e3-494d-afab-2b329cfb37fa)

**Reply by iamxiaojianzheng** [reply]
In addition, I repeated `<leader><space>` or other search operations to reopen the search window and then close it. The preview window is still there.

**Reply by dpetka2001** [reply]
This sounds similar to #3582 and was closed as something that should be reported in `fzf-lua` repo instead.
